tools/packaging/static-build/kernel/Dockerfile

```diff
@@ -23,4 +24,5 @@ RUN apt-get update && \
 	    rsync \
 	    cpio \
 	    patch && \
-    apt-get clean && apt-get autoclean
+    if [ "${ARCH}" != "$(uname -m)" ]; then apt-get install --no-install-recommends -y gcc-"${ARCH}"-linux-gnu binutils-"${ARCH}"-linux-gnu; fi && \
```

Comment:
Hi @jongwu, when building kernel for ARCH=ppc64le, this step would fail to find the packages
```
#5 41.60 E: Unable to locate package gcc-ppc64le-linux-gnu
#5 41.60 E: Unable to locate package binutils-ppc64le-linux-gnu
```
The correct packages are `gcc-powerpc64le-linux-gnu` and `binutils-powerpc64le-linux-gnu`. However, the ARCH should only be set as `powerpc64le` while building the image for the purpose of packages. When running it to build the kernel, the ARCH should be considered as `ppc64le` itself.